create/new_project/demos/erc20_btc/src/index.ts

```diff
@@ -173,3 +173,9 @@ async function printBalances(actor: Actor) {
 function toNominal(tokenWei: string, decimals: number) {
     return new BigNumber(tokenWei).div(new BigNumber(10).pow(decimals));
 }
+
+function wait_for_confirmation(text: string) {
+    if (process.env.NON_INTERACTIVE !== "true") {
```

Comment:
:wrench: For clarity; double negations are gnarly.

```suggestion
    if (process.env.INTERACTIVE == "true") {
```

Reply:
Yea, I was playing with this a bit... the problem is that this var can be undefined and then I still want to be it interactively. Let me push a different version though.